### PR TITLE
Alt lnpage

### PR DESCRIPTION
**Pull request by bodymindarts**

### CODE REVIEW

File: src/core/payment-status.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { toTypedError } from "@domain/utils"
+
+export const MakePaymentStatusChecker = ({
+  paymentRequest,
+}): Result<PaymentStatusChecker, LnInvoiceDecodeError> => {
+  const decodedInvoice = decodeInvoice(paymentRequest)
+  const getStatus = (): ResultAsync<PaymentStatus, PaymentStatusError> => {
+    return decodedInvoice
+      .asyncAndThen(({ paymentHash }) => findUserInvoice(paymentHash))
+      .map((invoice) => ({
```
Comment (nicolasburtey) [inline src/core/payment-status.ts:13]: Is there a need for an array here?

Comment (bodymindarts) [reply src/core/payment-status.ts:13]: This `.map` function is being called on the [`ResultAsync`](https://github.com/supermacro/neverthrow#resultasyncmap-method-1) type - its not an array.

File: src/core/payment-status.ts [hunk 2]
```diff
@@ -10,10 +10,7 @@ export const MakePaymentStatusChecker = ({
   const getStatus = (): ResultAsync<PaymentStatus, PaymentStatusError> => {
     return decodedInvoice
```
Comment (nicolasburtey) [inline src/core/payment-status.ts]: shouldn't it be plural `decodeInvoices`?

Comment (bodymindarts) [reply src/core/payment-status.ts]: this use case only operates on a _single_ invoice

**Pull request closed without merging**